posts/2021-02-15-eclipse-transformer.adoc

```diff
@@ -0,0 +1,506 @@
+|===
+
+(Source: https://github.com/OpenLiberty/open-liberty/issues/13393.)
+
+Here is the complete table, using only Java EE short names:
+
+.Java EE to Jakarta EE 9 Feature Updates: All Short Names
+|===
+| Java EE 8 feature name | Jakarta EE 9 feature name
+
+| appClientSupport-1.0
+| appClientSupport-2.0
+
+| appSecurity-3.0
+| appSecurity-4.0
+
+| batch-1.0
+| batch-2.0
+
+| batchManagement-1.0
+| batchManagement-2.0
+
+| beanValidation-2.0
+| beanValidation-3.0
+
+| cdi-2.0
+| cdi-3.0
+
+| concurrent-1.0
+| concurrent-2.0
+
+| ejb-3.2
+| ejb-4.0
+
+| ejbHome-3.2
+| ejbHome-4.0
+
+| ejbLite-3.2
+| ejbLite-4.0
+
+| ejbPersistentTimer-3.2
+| ejbPersistentTimer-4.0
+
+| ejbRemote-3.2
+| ejbRemote-4.0
+
+| el-3.0
+| el-4.0
+
+| jacc-1.5
+| jacc-2.0
+
+| jaspic-1.1
+| jaspic-2.0
+
+| javaMail-1.6
+| javaMail-2.0
+
+| javaee-8.0
+| jakartaee-9.0
+
+| javaeeClient-8.0
+| jakartaeeClient-9.0
+
+| jaxb-2.2
+| jaxb-3.0
+
+| jaxrs-2.1
+| jaxrs-3.0
+
+| jaxrsClient-2.1
+| jaxrsClient-3.0
+
+| jaxws-2.2
+| jaxws-3.0
+
+| jca-1.7
+| jca-2.0
+
+| jcaInboundSecurity-1.0
+| jcaInboundSecurity-2.0
+
+| jms-2.0
+| jms-3.0
+
+| jpa-2.2
+| jpa-3.0
+
+| jpaContainer-2.x
+| jpaContainer-3.0
+
+| jsf-2.3
+| jsf-3.0
+
+| jsfContainer-2.3
+| jsfContainer-3.0
+
+| jsonb-1.0
+| jsonb-2.0
+
+| jsonbContainer-1.0
+| jsonbContainer-2.0
+
+| jsonp-1.1
+| jsonp-2.0
+
+| jsonpContainer-1.1
+| jsonpContainer-2.0
+
+| jsp-2.3
+| jsp-3.0
+
+| managedBeans-1.0
+| managedBeans-2.0
+
+| mdb-3.2
+| mdb-4.0
+
+| servlet-4.0
+| servlet-5.0
+
+| wasJmsClient-2.0
+| wasJmsClient-3.0
+
+| wasJmsSecurity-1.0
+| wasJmsSecurity-3.0
+
+| wasJmsServer-1.0
+| wasJmsServer-3.0
+
+| webProfile-8.0
+| webProfile-9.0
+
+| websocket-1.1
+| websocket-2.0
+|===
+
+== Reference resources:
+
+Several resources within the Open-Liberty repository are useful as references:
+
+This project brings the Eclipse Transformer into the Open-Liberty environment, and provides the data set used within Open-Liberty:
+
+[source]
+--
+dev/wlp-jakartaee-transform
+--
+https://github.com/OpenLiberty/open-liberty/tree/integration/dev/wlp-jakartaee-transform
+
+The Open-Liberty transformation data set is located at:
+
+[source]
+--
+dev/wlp-jakartaee-transform/rules
+--
+https://github.com/OpenLiberty/open-liberty/tree/integration/dev/wlp-jakartaee-transform/rules
+
+Gradle tasks for using the Eclipse Transformer are located within:
+
+[source]
+--
+dev/wlp-gradle/subprojects/fat.gradle 
+--
+https://github.com/OpenLiberty/open-liberty/blob/integration/dev/wlp-gradle/subprojects/fat.gradle
+
+Java source for the the custom "repeat test action" which is used by Open-Liberty is located here:
+
+[source]
+--
+dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java
+--
+https://github.com/OpenLiberty/open-liberty/blob/integration/dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java
+
+In particular, the method "transformApp" shows a typical invocation of the transformer:
```

Comment:
You can point to a URL with a line number so that you can point right to the method.